tests/test_timing.py

```diff
@@ -0,0 +1,336 @@
+from dmutils import timing
+
+from collections import OrderedDict
+from contextlib import contextmanager
+from functools import lru_cache
+from itertools import product
+import json
+import logging
+import mock
+from numbers import Number
+import re
+import sys
+from time import sleep
+
+from flask import request
+import pytest
+
+
+class malleable_ANY:
+    def __init__(self, condition):
```

Comment:
You should probably explain here why this is useful.

Reply:
*Ah* - actually I wasn't going to because I only wanted this to be here temporarily before I promoted these to `-test-utils` where they *would* be documented properly.

Reply:
👍